tests/unit/cli/admin/test_import_artifacts.py

```diff
@@ -293,16 +293,18 @@ def test_import_artifacts(self):
         )
 
         args = [
-            "--api-server",
-            "http://127.0.0.1",
             "--db-uri",
             "postgresql://postgres:secret@127.0.0.1:5433",
             "--csv",
             "artifacts1of2.csv",
             "--csv",
             "artifacts2of2.csv",
         ]
-        result = invoke_command(import_artifacts.import_artifacts, [], args)
+        test_context["settings"].SERVER = "http://127.0.0.1"
+
+        result = invoke_command(
```

Comment:
You didn't answer my question earlier. Why do you use the helper when testing import_artifacts? The helper's main purpose seems to run invoke in an isolated_filesystem, save the result to a file, and return the contents.

This does not work for import_artifacts, and you even have to explicitly disable the behavior in the helper, when called for import_artifacts.

Reply:
I added a commit allowing the use `custom output` tests with `invoke_command`.
How does it look for you?